add missing include statement to fix compilation error for qjp2handler.cpp with recent JasPer:
"error: pow was not declared in this scope"
see also https://github.com/qt/qtimageformats/commit/704868db61be1542c2d9e2b75ead00c45c56cc36
--- qt-everywhere-src-5.14.2/qtimageformats/src/plugins/imageformats/jp2/qjp2handler.cpp.orig	2021-02-25 13:10:44.890921650 +0100
+++ qt-everywhere-src-5.14.2/qtimageformats/src/plugins/imageformats/jp2/qjp2handler.cpp	2021-02-25 13:11:04.310799662 +0100
@@ -45,6 +45,7 @@
 #include "qcolor.h"
 
 #include <jasper/jasper.h>
+#include <math.h> // for pow
 
 QT_BEGIN_NAMESPACE
 
